src/components/CallView/shared/EmptyCallView.vue

```diff
@@ -28,9 +29,15 @@
 </template>
 
 <script>
+import AccountMultipleIcon from 'vue-material-design-icons/AccountMultiple.vue'
```

Comment:
Grigorii started to follow this naming style, maybe let's keep it?
```suggestion
import IconAccountMultiple from 'vue-material-design-icons/AccountMultiple.vue'
```

Reply:
We were using "...Icon" ending too, which one to keep ?

Reply:
I'd migrate all later to have prefix `Icon...`, same as `Nc...` components. Then we don't need to group them later under `// Icons` comment

Reply:
"Icon*" follows a new convention on the server, groupware apps and libs. It also follows Bye style guide - related components should have a common prefix, not a suffix, so they are together alphabetically